Resources/public/css/views/subitem/list.css

```diff
@@ -116,11 +116,10 @@
     margin: 0;
 }
 
-.ez-view-subitemlistview .ez-subitem-priority-input {
+.ez-view-subitemlistview .ez-subitem-priority-form .ez-subitem-priority-input {
     padding: 0 1em;
     margin: 0;
     width: 100%;
-    height: 100%;
```

Comment:
Yes, I'm sure of the removal. This was giving a small vertical gap with the inputs and the priority buttons. Now that the input use the generic height: 1.8em defined for every inputs of the app, this is working perfectly.